documentation/staging/content/security/domain-security/weblogic-channels.md

```diff
@@ -13,9 +13,62 @@ unless absolutely necessary. If exposing any of these channels, limit access usi
 controls like security lists or set up a Bastion to provide access.
 {{% /notice %}}
 
-When accessing T3 or RMI based channels, the preferred approach is to `kubectl exec` into
-the Kubernetes Pod and then run `wlst`, or set up Bastion access and then run
-`wlst` from the Bastion host to connect to the Kubernetes cluster.
+When accessing T3 or RMI based channels for administrative purposes
+such as running WLST, the preferred approach is to `kubectl exec` into
+the Kubernetes Pod and then run `wlst.sh`, or set up Bastion access and then run
+`java weblogic.WLST` or `$ORACLE_HOME/oracle_common/common/bin/wlst.sh`
+from the Bastion host to connect to the Kubernetes cluster
+(some cloud environments use the term Jump Host or Jump Server instead of Bastion).
 
 Also, consider a private VPN if you need use cross-domain T3 access
 between clouds, data centers, and such.
+
+#### WebLogic HTTP channels
+
+When providing remote access to HTTP using a load balancer or similar,
```

Comment:
"or similar" is confusing here. I know that we mean something like an Ingress gateway, but it's not obvious here. It might be worth spelling it out a bit more such as "load balancer, Ingress gateway, or similar technology,"

Reply:
Done.